### DISCUSSION TITLE
Enabling contact tracer people

### DISCUSSION THREAD

**Discussion by LeCanardQuoi** [cds-snc/covid-alert-documentation#59] [Ideas] [answered]
The NZ Tracer leading covid-app claims "enabling contact tracing" via a "digital diary" (made up of timestamped QR code scanned contacts and timestamped BLE sensed contacts) for which the access to contact tracing people remains somehow controlled by the app user. Their successful app is deemed GAENS compliant and blessed by their privacy commissionner. Enabling timely contact tracers is part of the Kiwi strategy to hit early&hard.

As part of a stronger response to these more harmful strains, how can the #CovidAlert app do a better job at enabling contact tracing people poorly equipped across Canada? Can the NZ Tracer app proven best practices not be closely followed to do so??? At minimum, how the current app enables contact tracing (if it does) should be clarified.

**Comment by LeCanardQuoi** [comment]
Unclear what was the point to create a discussion section if nobody from HC, the advisory council, CDS product management staff, CDS Directors, provincial health organization or app core users is going to be motivated enough to participate on 1st discussion ever proposed, for instance. Ditto on moving enhancement requests from the issues section now dedicated to internal workflow matters. Not discussed anymore now than before. <edit'd for content, suggest you pick another phrase: obrien-j>

**Comment by obrien-j** [comment, marked as answer]
Morning @LeCanardQuoi ,

I'm not sure if you'd seen https://github.com/cds-snc/covid-alert-documentation/blob/main/product/qr-codes.md already, but as mentioned within, the COVID Alert teams are starting to research, design, and build the technical pieces of the service. Over the coming weeks they’ll post more detailed documentation on the functionality and flows. The work is happening in these repositories:

https://github.com/cds-snc/covid-alert-app
https://github.com/cds-snc/covid-alert-portal
https://github.com/cds-snc/covid-alert-server

As you mentioned, reducing burden on contact tracing efforts, and enabling more rapid venue based outbreak notifications are two of the goals of this work.

**Reply by LeCanardQuoi** [reply]
Thanks for participating. Have indeed reviewed and commented when 1st issued. Am far more concerned about timely enabling actual contact tracing people following the proven model of the Kiwi tracer app and the sharing of this "digital diary" under the control of the app user. Your researchers and designers would be better off imho following a proven working model six months ahead. Reinventing the wheel is unnecessary, certainly when there is urgency -vs- the 3rd wave VOCS.

**Reply by LeCanardQuoi** [reply]
It would appear that the Finns GAENS built app (no QR code scanning capability) is also being challenged for enabling contact tracers reqs: "data vital to contact tracking work is not provided to those exposed, nor to health care providers. [] All the information needed by trackers--like who, when and where--is not provided. It's kind of like if a car only had a single warning light for all of the vehicle's possible faults" https://yle.fi/uutiset/osasto/news/experts_claim_few_benefits_to_covid_tracing_app_paper_reports/11814681

**Reply by LeCanardQuoi** [reply]
UK NHS app is also realigning toward enabling contact tracing: "In the nearer term, a focus will be aligning technological operations more closely with the manual contact-tracing programme." https://www.publictechnology.net/articles/features/making-contact-inside-project-build-nhs-covid-19-app
